Handler/date-time.ts

```diff
@@ -0,0 +1,58 @@
+import * as isoly from "isoly"
+import { Converter } from "../Converter"
+import { Formatter } from "../Formatter"
+import { State } from "../State"
+import { StateEditor } from "../StateEditor"
+import { Settings } from "../Settings"
+import { add } from "./base"
+import { formatDate, stringToDate } from "./date"
+
+class Handler implements Converter<string>, Formatter {
+	toString(data: isoly.DateTime | any): string {
+		return typeof data == "string" ? data : ""
+	}
+	fromString(value: string): isoly.DateTime | undefined {
+		return stringToDate(value)
+	}
+	format(unformated: StateEditor): Readonly<State> & Settings {
+		let result = formatDate(unformated)
```

Comment:
Is the multiple if solution the only way?

Reply:
it's one way, not sure how to make it otherwise unless you makes nested stuff